2025-05-02 19:55:38,352 INFO    MainThread:99874 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-05-02 19:55:38,353 INFO    MainThread:99874 [wandb_setup.py:_flush():67] Configure stats pid to 99874
2025-05-02 19:55:38,353 INFO    MainThread:99874 [wandb_setup.py:_flush():67] Loading settings from /root/.config/wandb/settings
2025-05-02 19:55:38,353 INFO    MainThread:99874 [wandb_setup.py:_flush():67] Loading settings from /mnt/c/Users/Davy/Documents/Code/QM/CC/AudioLDM/wandb/settings
2025-05-02 19:55:38,353 INFO    MainThread:99874 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-05-02 19:55:38,353 INFO    MainThread:99874 [wandb_init.py:setup_run_log_directory():662] Logging user logs to ./wandb/run-20250502_195538-mrl3xilb/logs/debug.log
2025-05-02 19:55:38,362 INFO    MainThread:99874 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to ./wandb/run-20250502_195538-mrl3xilb/logs/debug-internal.log
2025-05-02 19:55:38,362 INFO    MainThread:99874 [wandb_init.py:init():781] calling init triggers
2025-05-02 19:55:38,362 INFO    MainThread:99874 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'project': 'qvim-clap-alignment', 'num_workers': 2, 'pin_memory': True, 'model_save_path': 'audioldm/qvim/models_clap_aligned', 'dataset_path': 'audioldm/qvim/data', 'audioldm_model': 'audioldm-m-full', 'clap_checkpoint': '/root/.cache/audioldm/clap/htsat-base.pt', 'clap_model': 'HTSAT-base', 'pretrained_name': 'mn10_as', 'random_seed': 42, 'continue_from': None, 'final_eval_dataset': 'val', 'val_split': 0.15, 'batch_size': 1, 'n_epochs': 100, 'early_stopping_patience': 10, 'early_stopping_min_delta': 0.0, 'weight_decay': 0.0003, 'max_lr': 0.0003, 'min_lr': 2.5e-05, 'warmup_epochs': 1, 'rampdown_epochs': 22, 'initial_tau': 0.07, 'tau_trainable': True, 'lr_schedule': 'cosine', 'debug': True, 'max_items': None, 'duration': 10.0, 'sample_rate': 32000, 'window_size': 800, 'hop_size': 320, 'n_fft': 1024, 'n_mels': 128, 'freqm': 8, 'timem': 300, 'fmin': 0, 'fmax': None, 'fmin_aug_range': 10, 'fmax_aug_range': 2000, 'output_dim': 512, '_wandb': {}}
2025-05-02 19:55:38,363 INFO    MainThread:99874 [wandb_init.py:init():809] starting backend
2025-05-02 19:55:38,363 INFO    MainThread:99874 [wandb_init.py:init():813] sending inform_init request
2025-05-02 19:55:38,369 INFO    MainThread:99874 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-02 19:55:38,369 INFO    MainThread:99874 [wandb_init.py:init():823] backend started and connected
2025-05-02 19:55:38,371 INFO    MainThread:99874 [wandb_init.py:init():915] updated telemetry
2025-05-02 19:55:38,449 INFO    MainThread:99874 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-05-02 19:55:39,323 INFO    MainThread:99874 [wandb_init.py:init():1014] starting run threads in backend
2025-05-02 19:55:39,417 INFO    MainThread:99874 [wandb_run.py:_console_start():2454] atexit reg
2025-05-02 19:55:39,418 INFO    MainThread:99874 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-05-02 19:55:39,418 INFO    MainThread:99874 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-05-02 19:55:39,418 INFO    MainThread:99874 [wandb_run.py:_redirect():2394] Redirects installed.
2025-05-02 19:55:39,420 INFO    MainThread:99874 [wandb_init.py:init():1056] run started, returning control to user process
2025-05-02 20:07:47,064 INFO    MsgRouterThr:99874 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
2025-05-02 20:07:48,719 ERROR   MainThread:99874 [redirect.py:_on_write():661] [no run ID] error in stdout callback
Traceback (most recent call last):
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 659, in _on_write
    cb(written_data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 2375, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 387, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 435, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 1518, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface.py", line 761, in publish_output_raw
    self._publish_output_raw(o)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 174, in send_record_publish
    self.send_server_request(server_req)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
2025-05-02 20:07:48,723 ERROR   MainThread:99874 [redirect.py:_on_write():661] [no run ID] error in stdout callback
Traceback (most recent call last):
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 659, in _on_write
    cb(written_data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 2375, in <lambda>
    lambda data: self._console_raw_callback("stdout", data),
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 387, in wrapper
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 435, in wrapper_fn
    return func(self, *args, **kwargs)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 1518, in _console_raw_callback
    self._backend.interface.publish_output_raw(name, data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface.py", line 761, in publish_output_raw
    self._publish_output_raw(o)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface_shared.py", line 38, in _publish_output_raw
    self._publish(rec)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/interface/interface_sock.py", line 39, in _publish
    self._sock_client.send_record_publish(record)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 174, in send_record_publish
    self.send_server_request(server_req)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 154, in send_server_request
    self._send_message(msg)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 151, in _send_message
    self._sendall_with_error_handle(header + data)
  File "/root/miniconda3/envs/qvim-baseline/lib/python3.10/site-packages/wandb/sdk/lib/sock_client.py", line 130, in _sendall_with_error_handle
    sent = self._sock.send(data)
BrokenPipeError: [Errno 32] Broken pipe
